src/components/MenuBars/MenuBars.tsx

```diff
@@ -110,74 +111,151 @@ export const MenuBars = ({showPreviousColumn, showNextColumn, onPreviousColumn,
-          disabled={!showPreviousColumn}
-          onClick={onPreviousColumn}
-          aria-hidden
-        >
-          <LeftArrowIcon className="menu-bars__navigation-icon" />
-        </button>
-      </section>
-
-      <section className={classNames("menu", "admin-menu", {"admin-menu--empty": !isAdmin, "menu-animation": animate})} onTransitionEnd={(event) => handleAnimate(event)}>
-        {isAdmin && (
+    <>
+      <aside className={classNames("menu-bars", {"menu-bars--admin": showAdminMenu, "menu-bars--user": !showAdminMenu}, {"menu-bars--isAdmin": isAdmin})}>
+        <section className={classNames("menu", "user-menu", {"menu-animation": animate})} onTransitionEnd={(event) => handleAnimate(event)}>
           <div className="menu__items">
-            <MenuButton tabIndex={TabIndex.AdminMenu} direction="left" label="Timer" onClick={showTimerMenu} icon={TimerIcon} />
-            <MenuButton tabIndex={TabIndex.AdminMenu + 1} direction="left" label="Voting" onClick={showVotingMenu} icon={VoteIcon} />
             <MenuToggle
-              value={state.moderation}
-              direction="left"
-              toggleStartLabel={t("MenuBars.startFocusMode")}
-              toggleStopLabel={t("MenuBars.stopFocusMode")}
-              icon={FocusIcon}
-              onToggle={toggleModeration}
-              tabIndex={TabIndex.AdminMenu + 2}
-              isFocusModeToggle
+              direction="right"
+              value={isReady}
+              toggleStartLabel={t("MenuBars.markAsDone")}
+              toggleStopLabel={t("MenuBars.unmarkAsDone")}
+              icon={CheckIcon}
+              onToggle={toggleReadyState}
+              tabIndex={TabIndex.UserMenu}
+            />
+            <MenuToggle
+              tabIndex={TabIndex.UserMenu + 1}
+              direction="right"
+              toggleStartLabel={t("MenuBars.raiseHand")}
+              toggleStopLabel={t("MenuBars.lowerHand")}
+              icon={RaiseHand}
+              onToggle={toggleRaiseHand}
+              value={raisedHand}
             />
+            <MenuButton tabIndex={TabIndex.UserMenu + 2} direction="right" label={t("MenuBars.settings")} onClick={showSettings} icon={SettingsIcon} />
           </div>
-        )}
 
+          <button
+            className={classNames("menu-bars__navigation", {"menu-bars__navigation--visible": showPreviousColumn || showNextColumn})}
+            disabled={!showPreviousColumn}
+            onClick={onPreviousColumn}
+            aria-hidden
+          >
+            <LeftArrowIcon className="menu-bars__navigation-icon" />
+          </button>
+        </section>
+
+        <section className={classNames("menu", "admin-menu", {"admin-menu--empty": !isAdmin, "menu-animation": animate})} onTransitionEnd={(event) => handleAnimate(event)}>
+          {isAdmin && (
+            <div className="menu__items">
```

Comment:
can this be a `ul` with each Button/Toggle being an `li`, just like the other menu? Would be better for a11y

Reply:
Agreed, I have refactored it.